### ISSUE TITLE
Which IDE used? step-by-step instructions?

### ISSUE THREAD

**Issue by manjunathgudisi** [saphanaacademy/SalesApp#1]
Hi
I would need to know which IDE you used to develop this project. Is it Web IDE Full-stack? Also, did you use CAP model? Can you summarize the (step-by-step) instructions to develop this project?
THanks, Manju

**Comment by saphanaacademy** [comment]
Hi Manju,
It accompanies this video tutorial which is part of the SAP HANA service series: 
https://www.youtube.com/watch?v=rElPNmMG--Y&list=PLkzo92owKnVyh290bEmKED11Gcdkyoi7w&index=21
It uses SAP HANA express edition so the IDE is SAP Web IDE for SAP HANA.
There's more context available in other videos in the playlist.
Thanks,
Philip
